psd-to-html-creative-agency/src/js/main.js

```diff
@@ -0,0 +1,11 @@
+const btn = document.querySelector(".hamburger");
+const navigation = document.querySelector(".navigation");
+const navLinks = document.querySelectorAll(".navigation__link");
+
+btn.addEventListener("click", () => {
+  navigation.classList.toggle("navigation--active")
+});
+
+navLinks.forEach(item => item.addEventListener("click", () => {
```

Comment:
`forEach` neveiks ant IE. Nors neatrodo, kad dabar čia kritinis funkcionalumas, vis tiek gerai būtų perdaryti, kad veiktų. Tikriausia prieš naudojant `forEach`paversti node list'ą į array'ų.

Reply:
Nepagalvojau apie tai, ir nezinojau kad IE neturi `.forEach` for `nodeList`